app/models/dqt/teacher_training.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: dqt_teacher_trainings
+#
+#  id                   :bigint           not null, primary key
+#  programme_end_date   :string
+#  programme_start_date :string
+#  programme_type       :string
+#  provider_ukprn       :string
+#  result               :string
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#  dqt_teacher_id       :bigint
+#
+# Indexes
+#
+#  index_dqt_teacher_trainings_on_dqt_teacher_id  (dqt_teacher_id)
+#
+module Dqt
+  class TeacherTraining < ApplicationRecord
```

Comment:
We already have a service `Dqt::RetrieveTrainingInstance` which (tries to 😅) identify the correct training instance for the trainee.

We should be consistent with our naming - are these training instances or are they teacher trainings?

I've always called them training instances cos i think that's what they were on DTTP but happy to go with whatever the consensus is on this!